CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.19.0 (19th June, 2021)
```

Comment:
@tomchristie should this be August 18th/19th (not sure your timezone)? I'm slightly surprised to see something mentiong a couple months ago, but it's been released ~9hours ago.

Reply:
🙃

Reply:
Oopsie — looks like that got merged without a change?

Reply:
Thanks for updating, I believe the GitHub release title can also be updated here https://github.com/encode/httpx/releases/tag/0.19.0

Reply:
Thanks @terencehonles!